build.xml

```diff
@@ -4,56 +4,59 @@
 	<property name="debug" value="true"/>
 	<property name="runtime" location="bin/ARE"/>
 	<property name="web.docroot" location="${runtime}/web/"/>
-	<property name="WebACS.bin" location="${web.docroot}/webapps/WebACS"/>
+	<property name="WebACS.bin" location="${web.docroot}/WebACS"/>
+	<property name="WebACS.version" value="v1.0.0"/>
```

Comment:
why do we have a static version of WebACS here? shouldn't it be just the latest release at every AsTeRICS release?

Reply:
like it was before, where we referenced to a defined version using git submodule, we also want to be sure the build is reproducible.
Using the latest version would result in two different builds when building today and in a half a year.

I will update the version of the WebACS as soon as the dynamic property feature is merged and released

Reply:
ok, makes sense! 👍